scripts/tests/test_flip_or_swap_gradients_axis.py

```diff
@@ -12,14 +12,15 @@
 
 
 def test_help_option(script_runner):
-    ret = script_runner.run('scil_flip_gradients.py', '--help')
+    ret = script_runner.run('scil_flip_or_swap_gradients_axis.py', '--help')
```

Comment:
we can come up with a better name: scil_gradients_reorder_axis.py something like this.
Also you forgot to delete test_swap_gradient_axis

Reply:
We could also use "bvecs" instead of "gradients". But if we want to follow the template of scil_"module_name", then scil_gradients_reorder_axis.py looks good. Should also rename `scilpy.io.gradient_table.py` as `scilpy.io.gradients.py` perhaps?

Reply:
But it does more than reorder, it flips too. Still ok? Any other idea?

Have we started using scil_module? Arnaud, final thoughts?

Reply:
> Also you forgot to delete test_swap_gradient_axis

Weird, I don't see it?